hdwallet-provider pulls dependency from github instead of npm

- [x] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

We [regularly have trouble pulling](https://app.circleci.com/jobs/github/smartcontractkit/chainlink/118687) `trufflesuite/provider-engine`, which you [have aliased](https://github.com/trufflesuite/truffle/blob/develop/packages/hdwallet-provider/package.json#L28) to `web3-provider-engine` for `@truffle/hdwallet-provider`. NPM and Yarn both maintain mirrors that help solve this problem. Could you publish your version of `trufflesuite/provider-engine` on NPM?

## Steps to Reproduce

Run `yarn install` from scratch on any repo that requires `truffle/hdwallet-provider`. We already [do this on our CI](https://app.circleci.com/jobs/github/smartcontractkit/chainlink/118686) and the builds that don't cache node packages fail regularly on this step. We have tried setting network concurrency to 1.

## Expected Behavior

`yarn`/`npm` installs work.

## Actual Results

```
yarn install v1.17.3
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error https://github.com/trufflesuite/provider-engine: Extracting tar content of undefined failed, the file appears to be corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
make: *** [yarndep] Error 1
```

## Environment

* Operating System: 
* Ethereum client: None
* Truffle version (`truffle version`): `hdwallet-provider@1.0.31`
* node version (`node --version`): `v10.16.3`
* npm version (`npm --version`): `yarn v1.17` up to `yarn v1.22`

> This is resolved in the latest Truffle (version 5.1.32).

I have same problme,

npm install @truffle/hdwallet-provider
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated eth-json-rpc-errors@1.1.1: Package renamed: https://www.npmjs.com/package/eth-rpc-errors
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t https://github.com/ethereumjs/ethereumjs-abi.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxx\AppData\Roaming\npm-cache\_logs\2020-07-02T07_24_46_061Z-debug.log


Truffle version (Truffle v5.1.32):
node version (v14.4.0):
npm version (6.14.5):

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@trufflesuite/eth-sig-util": "^1.4.2",
     "async": "^2.5.0",
     "backoff": "^2.5.0",
     "clone": "^2.0.0",
@@ -24,7 +25,6 @@
     "eth-json-rpc-filters": "^4.0.2",
     "eth-json-rpc-infura": "^3.1.0",
     "eth-json-rpc-middleware": "^4.1.1",
-    "eth-sig-util": "^1.4.2",
     "ethereumjs-block": "^1.2.2",
     "ethereumjs-tx": "^1.2.0",
     "ethereumjs-util": "^5.1.5",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@trufflesuite/web3-provider-engine",
-  "version": "14.0.7",
+  "version": "15.0.0-0",
   "description": "Fork of https://github.com/MetaMask/web3-provider-engine",
   "repository": "https://github.com/trufflesuite/provider-engine",
   "main": "index.js",

